File: bottlenote-product-api/src/main/java/app/Application.java
package app;

import java.util.TimeZone;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.domain.EntityScan;

@EntityScan(basePackages = "app")
@SpringBootApplication(scanBasePackages = "app")
public class Application {
  public static void main(String[] args) {
    TimeZone.setDefault(TimeZone.getTimeZone("Asia/Seoul"));
    SpringApplication.run(Application.class, args);
  }
}

File: bottlenote-admin-api/src/main/kotlin/app/Application.kt
package app

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.domain.EntityScan
import org.springframework.boot.runApplication
import java.util.*

@EntityScan(basePackages = ["app"])
@SpringBootApplication(scanBasePackages = ["app"])
class AdminApplication

fun main(args: Array<String>) {
    TimeZone.setDefault(TimeZone.getTimeZone("Asia/Seoul"))
    runApplication<AdminApplication>(*args)
}
